shared/src/main/scala/coop/rchain/store/LmdbKeyValueStore.scala

```diff
@@ -67,7 +66,9 @@ final case class LmdbKeyValueStore[F[_]: Sync](
   override def put[T](kvPairs: Seq[(ByteBuffer, T)], toBuffer: T => ByteBuffer): F[Unit] = {
     // Buffers for key and value created outside of transaction.
     // Why this helps (or why corruption happens) is not clear but this code will prevent corruption of the database.
-    val byteBuffers = kvPairs.map { case (key, value) => (key, toBuffer(value)) }
+    val byteBuffers = kvPairs.map {
+      case (key, value) => (key, toBuffer(value))
+    }
```

Comment:
We can map on tuples also so maybe this can be changed.
```scala
val byteBuffers = kvPairs.map(_.map(toBuffer))
```

Reply:
Nice